lib/helper/desktopHelper.py

```diff
@@ -38,8 +38,10 @@ def launch_browser(browser_type, **kwargs):
 
     if env.PROFILER_FLAG_CHROMETRACING in enabled_profiler_list:
```

Comment:
Agreed. I think @MikeLien can still choose if he wants to do this in his PR, or I will do another PR after this PR is in.